stable/airflow/values.yaml

```diff
@@ -490,11 +490,15 @@ postgresql:
     ## Enable PostgreSQL persistence using Persistent Volume Claims.
     enabled: true
     ##
-    ## Persistant class
+    ## Persistent class
     # storageClass: classname
     ##
-    ## Access mode:
-    accessMode: ReadWriteOnce
+    ## Access modes:
+    accessModes:
```

Comment:
ok for this, but can you rebase or rebuild all your Pull Request history? It seems like the merge commit mess up github.